src/dmd/mars.d

```diff
@@ -319,6 +319,10 @@ private int tryMain(size_t argc, const(char)** argv)
         {
             browse("http://dlang.org/dmd-openbsd.html");
         }
+        version (DragonFlyBSD)
+        {
+            browse("http://dlang.org/dmd-dragonflybsd.html");
```

Comment:
This doesn't exist, would rather you removed the nonexistent OpenBSD link above, as we don't put out official releases for these platforms.